packages/magiclink-auth-service-api/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:14-alpine
+ARG APP_NAME
+ARG BUILD_ID
+ARG VERSION
+WORKDIR /opt/app
+ENV APP_NAME="${APP_NAME}"
+ENV BUILD_ID="${BUILD_ID}"
+ENV VERSION="${VERSION}"
+ENV SERVER_PORT=3009
+# This must be built locally from the common package
+COPY --from=common:latest /opt/app ../common
+ADD src src
```

Comment:
If you move this to after add package-lock, I think it can improve build times. Docker caches parts of the build process, but has to rebuild after anything that changes, so best to put frequently changing elements at the end. Not 100% sure if this applies in this case, but might do.